roles/katello_repositories/tasks/main.yml

```diff
@@ -4,6 +4,3 @@
 
 - include_tasks: koji_repos.yml
   when: katello_repositories_use_koji
-
-- include_tasks: qpid.yml
-  when: ansible_distribution_major_version == "6" and katello_repositories_version == 3.2
```

Comment:
Useful for client testing on EL6

Reply:
Really? That `when` statement implies only Katello 3.2, which this PR removes.

Reply:
Ahh, it was needed for 3.2 on RHEL6, but it is also required for a client on EL6.